packages/grid/x-grid/src/tests/selection.XGrid.test.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { XGrid, useApiRef, ApiRef } from '@material-ui/x-grid';
+import { createClientRenderStrictMode } from 'test/utils';
+import { expect } from 'chai';
+import { getColumnHeaders } from 'test/utils/helperFn';
+
+describe('<XGrid /> - Reorder', () => {
```

Comment:
If you are happy with this approach, I think that it would make more sense to have the file into the `reordering.XGrid.test.tsx`.

Reply:
Yes it doesn't make sense in selection anymore.